src/vip-framework/checks/checkCorePoolComptroller.ts

```diff
@@ -84,8 +84,8 @@ export const checkCorePoolComptroller = () => {
       expect(await usdt.balanceOf(ACCOUNT)).to.gt(usdtBalance);
 
       const originalXVSBalance = await xvs.balanceOf(ACCOUNT);
-      expect(await comptroller["claimVenus(address)"](ACCOUNT)).to.be.not.reverted;
-      expect(await xvs.balanceOf(ACCOUNT)).to.be.gt(originalXVSBalance);
+      await expect(await comptroller["claimVenus(address)"](ACCOUNT)).to.be.not.reverted;
```

Comment:
Is not enough with the inner `await`?

Reply:
Removed the inner await https://github.com/VenusProtocol/vips/pull/195/commits/e1697b64705e249c2cfc3fc4f805be9ac5930af5